### DISCUSSION TITLE
cfg file support?

### DISCUSSION THREAD

**Discussion by ninjamonkey198206** [timonwong/vscode-shellcheck#657] [Q&A] [answered]
Trying to get vscode and this extension to recognize the cfg files included in the primary sh file, and I'm not sure how. Any ideas?

**Comment by felipecrs** [comment, marked as answer]
Set the VS Code file association for your files to be `shellscript`. Like:

```json
"files.associations": {
  "*.cfg": "shellscript"
}
```

**Reply by ninjamonkey198206** [reply]
Ah, thank you! That didn't even cross my mind.
Worked perfectly.
